modules/primitives/src/lib.rs

```diff
@@ -61,22 +61,20 @@ impl Convert<Price, Balance> for BalancePriceConverter {
 bitmask! {
 	#[derive(Encode, Decode, Default)]
 	pub mask Leverages: u16 where flags Leverage {
-		LongOne 	= 0b0000000000000001,
-		LongTwo 	= 0b0000000000000010,
-		LongThree 	= 0b0000000000000100,
-		LongFive 	= 0b0000000000001000,
-		LongTen 	= 0b0000000000010000,
-		LongTwenty 	= 0b0000000000100000,
-		LongThirty	= 0b0000000001000000,
-		LongFifty	= 0b0000000010000000,
-		ShortOne 	= 0b0000000100000000,
-		ShortTwo 	= 0b0000001000000000,
-		ShortThree 	= 0b0000010000000000,
-		ShortFive 	= 0b0000100000000000,
-		ShortTen 	= 0b0001000000000000,
-		ShortTwenty = 0b0010000000000000,
-		ShortThirty	= 0b0100000000000000,
-		ShortFifty	= 0b1000000000000000,
+		LongTwo 	= 0b0000000000000001,
+		LongThree 	= 0b0000000000000010,
+		LongFive 	= 0b0000000000000100,
+		LongTen 	= 0b0000000000001000,
+		LongTwenty 	= 0b0000000000010000,
+		LongThirty	= 0b0000000000100000,
+		LongFifty	= 0b0000000001000000,
+		ShortTwo 	= 0b0000000010000000,
```

Comment:
```suggestion
		ShortTwo 	= 0b0000000100000000,
```

I want first 8 bits for long and last 8 bits for short so short to long is `>> 8` and long to short is `<< 8`

Reply:
@xlc ok

Reply:
@xlc updated tests to make sure you never it covers everything